### PR TITLE
Bike

### PR DESCRIPTION
**Pull request by burovetskyi**
[DEMO LINK](https://burovetskyi.github.io/layout_miami/)

### CODE REVIEW

**Comment by burovetskyi** [comment]
> [DEMO LINK](https://burovetskyi.github.io/layout_miami/)

File: src/styles/blocks/menu.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
+@import "../utils/variables";
+@import "../utils/mixins";
+
+.menu {
```
Comment (anna-inozemets) [inline src/styles/blocks/menu.scss:4]: https://prnt.sc/L3Ekd4-d3EEY => Just add z-index and position to fix it)

https://prnt.sc/48UHyMaZ20aU => add cross icon (I can click, but I can't see an icon)

File: src/styles/blocks/logo.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+@import "../utils/mixins";
+
+.logo {
```
Comment (anna-inozemets) [inline src/styles/blocks/logo.scss:3]: https://prnt.sc/05w4yb6Ogr4o => Add correct href atribute in html, to fix it (It should be "#", not a "/"

File: src/styles/blocks/button.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
+@import "../utils/variables";
+@import "../utils/mixins";
+
+.button {
```
Comment (anna-inozemets) [inline src/styles/blocks/button.scss:4]: https://prnt.sc/NkE4D6PAZlFK => button for form has border, I think it is better to do all button in the same style

**Review by anna-inozemets — COMMENTED** [review]
Pretty good job, just fix smth!)

**Pull request closed without merging**